app/Config/Logger.php

```diff
@@ -109,7 +109,7 @@ class Logger extends BaseConfig
 			'filePermissions' => 0644,
 
 			/*
-			 * Logging Directory Path
+			 * Error Logging Directory Path
```

Comment:
Except sometimes the message isn't an error.

Reply:
resolved.